playbooks/roles/edxapp/tasks/service_variant_config.yml

```
@@ -165,7 +165,7 @@
     state: link
     force: yes
   become_user: "{{ supervisor_user }}"
-  when: celery_worker is not defined and not disable_edx_services and
+  when: celery_worker is defined and not disable_edx_services and
```

I hate `is defined` check. It means nothing. Please choose something more readable such as `celery_worker|default(True)`.

```suggestion
  when: celery_worker|default(true) and not disable_edx_services and
```

---

To be honest, I prefer to make this change in a separate PR, Ansible conditionals and vars are messy sometimes, or at least for me. This works, and the check is done in this way in more than one place, testing this is time consuming, so I prefer to make the change when I'm sure I have the time to test it.